assets/src/edit-story/elements/text/updateForResizeEvent.js

```diff
@@ -22,17 +22,24 @@ import {
   calculateTextHeight,
 } from '../../utils/textMeasurements';
 import { dataPixels } from '../../units';
+import { resizeRules } from './index';
 
-function updateForResizeEvent(element, direction, newWidth, newHeight) {
+function updateForResizeEvent(element, direction, newWidth, newHeight, delta) {
```

Comment:
Should we unit-test this function? It does quite a bit of heavy-lifting now. We should at least document it with good JSDoc here.

Reply:
Added JSDoc. We have some coverage in Karma tests and I think that's the way with Text element (and measurer).